packages/react/src/accordion/Accordion.tsx

```diff
@@ -15,7 +15,7 @@ export const Accordion = forwardRef<HTMLDivElement, AccordionProps>(
     const { restProps, sprinkleProps } = extractSprinkles(props);
 
     return (
-      <Box asChild borderB="1" ref={ref} {...sprinkleProps}>
+      <Box asChild ref={ref} w="320" {...sprinkleProps}>
```

Comment:
I think width should be passed from the stories if needed; if not passed it would take the whole container.
```suggestion
      <Box asChild ref={ref} {...sprinkleProps}>
```